src/SFA.DAS.EmployerAccounts.Web/ViewModels/CallToActionViewModel.cs

```diff
@@ -29,5 +27,6 @@ public class CallToActionViewModel
         public string HashedDraftApprenticeshipId { get; set; }
         public string HashedCohortReference { get; set; }
         public string ApprenticeName { get; set; }
+        public bool ViewApprenticeDetails => CourseStartDate.HasValue && CourseEndDate.HasValue;
```

Comment:
What do these two properties have to do with "View Apprentice Details"?

Reply:
> What do these two properties have to do with "View Apprentice Details"?

WithProvider status has two link "Add more details" or "View Apprentice details".. we dont need this any more.. if its with provider then  they can only View not Edit